src/main/resources/static/js/js_utils.js

```diff
@@ -26,13 +26,13 @@ function handleAlerts(id){
     }
 }
 
-function copy2clipbord(id) {
+function copy2clipbord(id, dontShowAlert) {
     let copyText = document.getElementById(id);
 
     navigator.clipboard.writeText(copyText.value).then(function() {
-        handleAlerts("alert-success-copy");
+        if(dontShowAlert === false) handleAlerts("alert-success-copy");
```

Comment:
not a good practive to use double negation since it's not easy to read.

use "showAlert"

if (showAlert === true)